examples/02-Merlin-Models-and-NVTabular-integration.ipynb

```diff
@@ -78,9 +78,12 @@
      "name": "stderr",
```

Comment:
<p>How/where should I execute these notebooks to remove my personal home directory from the results?</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/NVIDIA-Merlin/models/pull/813/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='4'/>